build.proj

```diff
@@ -111,11 +141,12 @@
       Condition="!Exists('$(ZipExe)')" />
   </Target>
   
-  <Target Name="Build" DependsOnTargets="BuildMsBuildTask;PrepareForAutoRestLibraries;RestoreNugetPackages">
+  <Target Name="Build" DependsOnTargets="BuildClientRuntime;BuildMsBuildTask;PrepareForAutoRestLibraries;RestoreNugetPackages">
     <PropertyGroup>
       <_ExtraPropertyList>CodeSign=$(CodeSign)</_ExtraPropertyList>
       <_TemporaryNetCoreFeeds>-s https://api.nuget.org/v3/index.json -s https://dotnet.myget.org/F/cli-deps/api/v3/index.json</_TemporaryNetCoreFeeds>
     </PropertyGroup>
+    <CallTarget Targets="DisableSN" Condition=" '$(OS)' == 'Windows_NT'"/> 
```

Comment:
It's needed because one of the test projects uses an assembly that's not actually strong named, we had to do this in the AutoRest project to support that too.

The actual release binaries still get validated during the release process.